.github/workflows/tf_apply.yml

```diff
@@ -7,21 +7,23 @@ on:
 permissions: {}
 
 env:
-  TF_VAR_scw_organization_id: ${{ secrets.SCW_ORGANIZATION_ID }}
-  TF_VAR_scw_access_key: ${{ secrets.SCW_TF_CI_ACCESS_KEY }}
-  TF_VAR_scw_secret_key: ${{ secrets.SCW_TF_CI_SECRET_KEY }}
+  CI: true
```

Comment:
A priori déjà présent par défaut: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables